packages/create-starter/src/index.ts

```diff
@@ -100,16 +107,22 @@ async function createStarter(options: prompts.Answers<'name' | 'kit'>): Promise<
       await initNodeProject(packageJsonPath, destPath, options);
     }
 
-    await initGitRepo(destPath);
+    await initGitRepo(destPath, packageManager);
     console.log(bold(green('✔') + ' Done!'));
     console.log('\nNext steps:');
     console.log(` ${bold(cyan(`cd ${options.name}`))}`);
```

Comment:
I think Dustin wanted to update the language here to be this based on the slack thread disucssion:

oh update next steps to say "npm start" (or equivalent based on package manager selected" instead of "npm install"

I double check with him on that.

Reply:
When the kit is created we would still be outside of the newly created directory though and would still need to cd into it.